app/models/course/group_user.rb

```diff
@@ -0,0 +1,12 @@
+class Course::GroupUser < ActiveRecord::Base
+  stampable
+  belongs_to :user, inverse_of: :group_users
```

Comment:
Add a validation that the user added belongs to the course which the group belongs.